### ISSUE TITLE
Add binary for Darwin/Intel architecture

### ISSUE THREAD

**Issue by caioariede** [sbdchd/squawk#355]
Right now the `x86` architecture is actually ARM, not Intel.

```
☁  ~  file ~/Downloads/squawk-darwin-x86_64
/Users/caio/Downloads/squawk-darwin-x86_64: Mach-O 64-bit executable arm64
```

**Comment by chdsbd** [comment]
Thanks for the report @caioariede!

This is fixed in v1.1.1: https://github.com/sbdchd/squawk/releases/tag/v1.1.1

We have linux x64, macOS x64 and macOS amd64

**Comment by caioariede** [comment]
Thank you!
